client/components/Problem/Problem.js

```diff
@@ -32,7 +33,7 @@ export class Problem extends Component {
                 <div className="answers">
                     <span>{problem.answersCount} answers</span>
                 </div>
-                <div className="view-button">
+                <div className="view-button" onClick={() => viewButtonClick()}>
```

Comment:
better to use <button/> if you really have a button.
for now it is ok to have view-button selector - but it is very abstract - and can cause collisions later, when someone will name his button view-button too.
better to apply BEM naming style: for example problem-view-answers-button

but I think that in you app there will be COMMON styled buttons, for example:
 btn-primary btn-error btn-warning btn-transparent etc. in such cases best idea is to create global button styles.